### PR TITLE
fix: can't stop dev on component after switching to a different project

### LINKED ISSUE #537: Can't stop dev on component after switching to a different project
Steps to reproduce:
1. Create two projects
2. In one of the projects create new component and start dev on it
3. Switch to a different project and right back to the first project using the k8s extension
4. The component is still in dev mode but does not have option to stop the dev mode

### LINKED ISSUE #634: Refreshing view makes me unable to stop the dev on component
When I start the dev on cluster and refresh the openshift view, the component no longer offers to stop the dev.

https://github.com/redhat-developer/intellij-openshift-connector/assets/86789336/cdd8c99f-7bf2-4357-bcdf-9b992effc5bc

### PR DESCRIPTION
**Pull request by sbouchet**
## Was the change discussed in an issue?
fixes #537 
fixes #634

### CODE REVIEW

**Comment by sbouchet** [comment]
@adietich: i've done some quite big refactor, and not very happy with some method signatures on odo interface. 
WDYT ?

**Comment by sbouchet** [comment]
this will also address some of  #169

**Review by adietish — APPROVED** [review]

File: src/main/java/org/jboss/tools/intellij/openshift/actions/service/DescribeServiceAction.java [hunk 1]
```diff
@@ -1,53 +0,0 @@
...
- * Copyright (c) 2019-2020 Red Hat, Inc.
- * Distributed under license by Red Hat, Inc. All rights reserved.
- * This program is made available under the terms of the
- * Eclipse Public License v2.0 which accompanies this distribution,
- * and is available at http://www.eclipse.org/legal/epl-v20.html
- *
- * Contributors:
- * Red Hat, Inc. - initial API and implementation
- ******************************************************************************/
-package org.jboss.tools.intellij.openshift.actions.service;
```
[the remark this answers is not in the document]
Comment (sbouchet) [reply src/main/java/org/jboss/tools/intellij/openshift/actions/service/DescribeServiceAction.java:11]: describe service with odo has been removed in v3 https://odo.dev/docs/user-guides/v3-migration-guide/#commands-added-modified-or-removed-in-v3

**Pull request merged**